client/lib/abtest/active-tests.js

```diff
@@ -136,4 +141,14 @@ export default {
 		localeTargets: 'any',
 		countryCodeTargets: [ 'JP' ],
 	},
+	[ RUM_DATA_COLLECTION.AB_NAME ]: {
+		datestamp: '20200602',
+		variations: {
+			[ RUM_DATA_COLLECTION.AB_VARIATION_ON ]: 5,
+			[ RUM_DATA_COLLECTION.AB_VARIATION_OFF ]: 95,
+		},
+		defaultVariation: RUM_DATA_COLLECTION.AB_VARIATION_OFF,
+		localeTargetes: 'any',
```

Comment:
typo: `localeTargets`. If the code was left as-is, only data for `en` locales would be collected -- that's the default.

Reply:
🤦 Good catch, thanks!